include/circt/Dialect/FIRRTL/FIRRTLStatements.td

```diff
@@ -66,8 +67,7 @@ def StrictConnectOp : FIRRTLOp<"strictconnect", [SameTypeOperands, FConnectLike]
 
   let hasVerifier = 1;
 
-  let assemblyFormat =
```

Comment:
Can you use a custom directive instead of full customer parser/printer?  https://mlir.llvm.org/docs/DefiningDialects/Operations/#custom-directives

Reply:
Done, maybe we can use `OptionalConnectOperandTypes` for normal connects as well.